sycl/include/CL/sycl/access/access.hpp

```diff
@@ -45,7 +45,9 @@ enum class address_space : int {
   private_space = 0,
   global_space,
   constant_space,
-  local_space
+  local_space,
+  global_device_space,
+  global_host_space
```

Comment:
Are these two mapped into mere `global_space` for non-FPGA?

Reply:
It depends on what level we are talking about. From C++/SYCL perspective - not. On LLVM IR level - it depends on a triple we set, if it's not SPIR - it's going to be mapped on global address space and to device/host space otherwise. On SPIR-V level it's mapped on CrossWorkgroup storage class (aka global address space) for non-FPGA targets.